### PR TITLE
LOC 7434

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Locations for Enda Yohannes

### CODE REVIEW

File: new/LOC7434HaddushAddi.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2024-01-24">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="tabiya">
```
Comment (eu-genia) [inline new/LOC7434HaddushAddi.xml:47]: can you, when creating a place or institution, also create a wikidata entry, and link the two? thank you!

File: new/LOC7435MayKwak.xml [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                <place type="qushat">
+                    <placeName xml:id="n1" xml:lang="ti">ማይ፡ ኳክ፡</placeName>
+                    <placeName corresp="#n1" xml:lang="en">Māy Kʷāḵ</placeName> 
+                    <region notAfter="1991" ref="LOC6569Tegray" type="province"/>                    
+                    <settlement when="2024-01-09" ref="LOC7434HaddushAddi" type="tabiya">ṭābiyā Ḥādduš ʿAddi</settlement>
+                    <country ref="LOC3010Ethiop"></country>
+                </place>
+            </listPlace>
+            <listRelation>                        
+                <relation name="skos:exactMatch" active="LOC7435MayKwak" passive="INS1008EndaYohannes"/>
```
Comment (eu-genia) [inline new/LOC7435MayKwak.xml:56]: it seems to me that `qushat `cannot be identical to `INS`, only a `church `/ `monastery `can be. if LOC7435 is the district then you can locate INS1008 in the district. 
You may (you do not have to) create a `LOC `for the church too, then these would be linked by the `exactMatch `relation

File: new/LOC7435MayKwak.xml [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2024-01-24">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="qushat">
```
Comment (eu-genia) [inline new/LOC7435MayKwak.xml:47]: wd: as above is always desirable

File: new/LOC7434HaddushAddi.xml (hunk 1 above)
Comment (eu-genia) [reply new/LOC7434HaddushAddi.xml:47]: see also https://github.com/BetaMasaheft/Documentation/issues/1896

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LOC7434HaddushAddi.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/LOC7434HaddushAddi.xml:47]: I see in the issue that there is a proposal to give this task to students.

File: new/LOC7435MayKwak.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/LOC7435MayKwak.xml:47]: S. above.

File: new/LOC7435MayKwak.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/LOC7435MayKwak.xml:56]: I did it after `<relation name="skos:exactMatch" active="INS1007Taqot_Maryam" passive="LOC7409Taqot"/>`. Isn't Enda Yohannes an institution? It stands in the place (qushet) May Kwak, but somehow this name is not attached to the name of the church.

Comment (DenisNosnitsin1970) [reply new/LOC7435MayKwak.xml:56]: Here, there is no qushet Enda Yohannes.

File: new/LOC7434HaddushAddi.xml (hunk 1 above)
Comment (eu-genia) [reply new/LOC7434HaddushAddi.xml:47]: Students should go through the old records; when creating new ones we must think about `wd:` according to the guidelines.

File: new/LOC7435MayKwak.xml (hunk 2 above)
Comment (eu-genia) [reply new/LOC7435MayKwak.xml:56]: District is a district, church is a church, they are not identical. We need exactMatch only if we have a LOC record for the church itself, and the church is also a repository and thus an INS. Then the very same things can be linked by exactMatch.

Comment (eu-genia) [reply new/LOC7435MayKwak.xml:56]: Remove the `relation `here and under Enda Yohannes. It is enough that the qushat is under `settlement `there

**Review by eu-genia — APPROVED** [review]

**Pull request merged**